runtime/src/accounts_db.rs

```diff
@@ -451,11 +451,40 @@ pub struct AccountsDB {
     /// Thread pool used for par_iter
     pub thread_pool: ThreadPool,
 
+    pub thread_pool_clean: ThreadPool,
+
     /// Number of append vecs to create to maximize parallelism when scanning
     /// the accounts
     min_num_stores: usize,
 
     pub bank_hashes: RwLock<HashMap<Slot, BankHashInfo>>,
+
+    pub dead_slots: RwLock<HashSet<Slot>>,
+}
+
+fn make_min_priority_thread_pool() -> ThreadPool {
+    use thread_priority::{
+        set_thread_priority, thread_native_id, NormalThreadSchedulePolicy, ThreadPriority,
+        ThreadSchedulePolicy,
+    };
+    let num_threads = get_thread_count();
+    rayon::ThreadPoolBuilder::new()
+        .spawn_handler(|thread| {
+            std::thread::spawn(|| {
+                let thread_id = thread_native_id();
+                set_thread_priority(
+                    thread_id,
+                    ThreadPriority::Min,
+                    ThreadSchedulePolicy::Normal(NormalThreadSchedulePolicy::Normal),
+                )
+                .unwrap();
+                thread.run()
+            });
+            Ok(())
+        })
+        .num_threads(num_threads)
+        .build()
```

Comment:
Related to [this](https://github.com/solana-labs/solana/pull/8799/files#r396204269), how about `.name("solana-...")`-ing?